packages/xml-views-tooltip/src/tooltip.ts

```diff
@@ -108,11 +108,15 @@ function splitQNameByNamespace(
   const matchGroups = match!.groups!;
   return {
     ns: matchGroups.ns,
-    name: matchGroups.name ?? "",
+    name:
+      matchGroups.name ??
+      /* istanbul ignore next */
+      "",
   };
 }
 
 function elementClosingTagToFQN(xmlElement: XMLElement): string {
+  /* istanbul ignore next */
```

Comment:
Add a comment that the closeName can't be undefined here because otherwise the ast position visitor wouldn't return its type